tests/SlicerTestSuite.hs

```diff
@@ -11,11 +11,15 @@ tests :: TestTree
 tests =
     testGroup "Run TML files" (map runTMLTest
       [ "abs"
+      , "array"
```

Comment:
One thing to remember is that new examples should be added to a list of tests AND to a list of benchmarks in `bench` directory. I will fix these now, so no worries - just something to remember in the future :-)  I know this duplication is irritating but I don't see a way of avoiding it.